content/docs/integrations/kubernetes/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Adding a Kubernetes cluster
+lastUpdated: '2021-07-15T21:00:00.0Z'
+description: How to add a Kubernetes cluster for the Kubernetes plugin.
+---
+
+![Full active cluster on roadie](./active.png)
+
+> ⚠️ Note, the Kubernetes plugin is only supported with AWS clusters. GKE support is coming soon.⚠️
+
+# AWS
+
+## Introduction
+
+In order to use the Kubernetes plugin, Roadie needs:
+ * An Assumed Role to fetch the resources from your cluster
+ * The name of your cluster
+ * URL of your Kubernetes API Server endpoint
+
+
+These are set within Roadie at the following url:
+
+```text
+https://<tenant-name>.roadie.so/administration/settings/kubernetes
+```
+
+This page describes how to create and set up the API token.
+
+## Steps
+
+In this section we will create an AWS role that will allow the roadie user to access (read only) your Kubernetes cluster's resources.
+ * [Assuming Role](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)
```

Comment:
It's not obvious where these links go until I click them. I would say something like "You can learn more about these concepts in the AWS docs:"